app/controllers/addPaymentTypeCtrl.js

```diff
@@ -0,0 +1,29 @@
+'use strict'; 
+const prompt = require('prompt');
+
+module.exports.promptPaymentType = () => {
+  return new Promise((resolve, reject) => {
+    prompt.get([
+      {
+        name: "payment",
+        description: `Please enter one of the following payment methods (AmEx, MasterCard, Visa, PayPal)`,
+        type: 'string',
+        pattern: /^AmEx|MasterCard|Visa|PayPal$/,
+        message: 'Please enter one of the available options exactly as it appears',
+        required: true
+      },
+      {
+        name: "accountNumber",
+        description: `Enter account number`,
+        type: 'integer',
```

Comment:
When I try to input a long string of numbers (like the length of a credit card number) I am told it must be an integer. When I put in a shorter number ('222') it accepted it. Look into types a bit to see if you can get it to accept bigger integers.

Reply:
Niceeeee, changed 'type' to 'number' in prompt